### ISSUE TITLE
Feature request: Aggregation store - reload models in Hjson config files at runtime without restarting application

### ISSUE THREAD

**Issue by thaingo** [yahoo/elide#2264]
Currently any model change in Hjson config files does require an application restart so that the change will be effective. Possibility to reload models in Hjson config files at runtime would make analytic features more appealing and convenient for Elide users.

**Comment by aklish** [comment]
:+1:

We have the same requirement for Yavin.  It is on the list of things we hope to do soon.

**Comment by aklish** [comment]
This isn't the long term solution, but I found this works nicely for editing HJSON without restarting the service:

https://github.com/yahoo/elide-spring-boot-example/pull/40

**Comment by thaingo** [comment]
Thanks @aklish . Agreed. 

FYI, I have tried spring boot dev-tools before filing this issue.

I also tried another approach [using this WatchService](https://docs.oracle.com/javase/tutorial/essential/io/notification.html), then reload respective bean but there are some drawbacks like:

1. Not working if your file system is distributed one
2. The solution is not elegant. Reloading a bean is not that transparent and straightforward as it is documented.
